libraries/AP_GPS/AP_GPS_SBF.cpp

```diff
@@ -453,8 +453,10 @@ AP_GPS_SBF::process_message(void)
             set_alt_amsl_cm(state, ((float)temp.Height - temp.Undulation) * 1e2f);
         }
 
-        if (temp.NrSV != 255) {
-            state.num_sats = temp.NrSV;
+	state.num_sats = temp.NrSV;
```

Comment:
correct indentation

Reply:
The indentation has been corrected, thank you for catching that